opm/parser/eclipse/EclipseState/Schedule/Well.cpp

```diff
@@ -212,6 +212,10 @@ namespace Opm {
         m_completions->add( time_step , newCompletionSet);
     }
 
+    void Well::addCompletionsSet(size_t time_step, const CompletionSetConstPtr newCompletionSet){
```

Comment:
The `addCompletions()` method above should use the `addCompletionsSet()` (I suggest `addCompletionSet()`) method.